hangman_file.js

```diff
@@ -0,0 +1,74 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Game is working decently although not flawlessly since I noticed glitches with lives count, and also gussedWords. Also it lacks and graphical indicator as to how many lives are remaining. Please continue to work on this on your free time since it's important to practice debugging and working with HTML image tags. Keep it up and looking forward to se finished version in the future.